doc/guides/snapshot.md

```diff
@@ -59,7 +59,7 @@ The below snippet will download and extract the snapshot inline. This may be ben
 ```bash
 wget -O latest.txt https://pocket-snapshot.liquify.com/files/latest.txt
 latestFile=$(cat latest.txt)
-wget -c "https://pocket-snapshot.liquify.com/files/$latestFile" -O - | sudo tar -xz -C {POCKET_DATA_DIR}
+wget -c "https://pocket-snapshot.liquify.com/files/$latestFile" -O - | sudo tar -xv -C {POCKET_DATA_DIR}
```

Comment:
Can you add more changes like this?

```shell
[ -d ${POCKET_DATA_DIR} ] || mkdir -p ${POCKET_DATA_DIR}
wget -c "https://pocket-snapshot.liquify.com/files/$(curl -s https://pocket-snapshot.liquify.com/files/latest.txt)" -O - \
  | tar -xv -C ${POCKET_DATA_DIR}
```

- Missing dollar sign in front of {POCKET_DATA_DIR}
- No need to download latest.txt as a file

Reply:
Happy to do that, I'll push changes in a bit. 

By the way, I've been working on a script that consolidates some of the steps here and in the OPERATE A NODE docs, would love to get your take on it if you're interested: https://github.com/0xThresh/pokt-node-setup/blob/pulumi-demo/node-setup.sh

Reply:
> By the way, I've been working on a script that consolidates some of the steps here and in the OPERATE A NODE docs, would love to get your take on it if you're interested: https://github.com/0xThresh/pokt-node-setup/blob/pulumi-demo/node-setup.sh

Thank you for making this effort. I like this approach with a single script.  Why don't you integrate your script to [the top page](https://github.com/pokt-network/docs-v2/blob/main/access-tutorials/README.md) of the tutorial?  You can simply add a new section like "Quick script to set up a new node" above or below the "Manual Setup Tutorial" section.

Reply:
> Thank you for making this effort. I like this approach with a single script. Why don't you integrate your script to [the top page](https://github.com/pokt-network/docs-v2/blob/main/access-tutorials/README.md) of the tutorial? You can simply add a new section like "Quick script to set up a new node" above or below the "Manual Setup Tutorial" section.

Glad to hear you like it! I have a bit more testing to do before I submit that; the nginx server isn't coming up correctly yet and I need to add pocket accounts. I'll keep working on it and submit a new PR once I get further along. 

I'm also wondering how much of this stays relevant with Shannon. Do you know if there are big changes to the node setup process for Shannon?